### PR TITLE
daemon: improve debugging line

### PR DESCRIPTION
**Pull request by leseb**
When setting DEBUG=verbose, this only activates bash tracing with set
-x. This patch adds more info during the debug like the name of the
script and the line being executed. See:

$ ./a.sh
+./a.sh:6: echo a
a

Signed-off-by: Sébastien Han <seb@redhat.com>

### CODE REVIEW

File: ceph-releases/jewel/ubuntu/14.04/daemon/debug.sh [hunk 1]
```diff
@@ -11,6 +11,8 @@ for option in $(comma_to_space ${DEBUG}); do
   case $option in
     verbose)
       echo "VERBOSE: activating bash debugging mode."
+      DEBUG_MS=20
+      echo "VERBOSE: Ceph daemon will now run in debugging level $DEBUG_MS"
```
Comment (liewegas) [inline ceph-releases/jewel/ubuntu/14.04/daemon/debug.sh]: This is unlikely to be the ceph debugging the user needs.  I would suggest instead having a different env var CEPH_ARGS that is passed through, independent of $VERBOSE?

Comment (leseb) [inline ceph-releases/jewel/ubuntu/14.04/daemon/debug.sh]: What kind of debug options can we pass to CEPH_ARGS? Is that going to help running daemon with a higher debugging level?

**Comment by liewegas** [comment]
anythin you pass on the command line (e.g., CEPH_ARGS="--debug-ms 1 --debug-osd 10")

**Comment by leseb** [comment]
@liewegas alright thanks, will rework this then.

**Pull request merged**